DESCRIPTION

```diff
@@ -31,21 +32,28 @@ Imports:
     yaml,
     here,
     crayon,
+    callr,
     progress,
+    pingr,
     forcats,
     assertthat,
     rstudioapi,
     stringr,
-    shiny,
+    shiny (>= 1.6.0),
```

Comment:
@barrettk I was talking with @andersone1 about this: we may want to move all of these new packages (the Shiny ones, and also `callr`, etc.) to `Suggests` and then gate the `v()` function (something like [this](https://github.com/metrumresearchgroup/bbr.bayes/blob/710ed177c504b5e378957fd9a0812e4fb1ccb5b8/R/submit-model.R#L31)) so that it can only be used if they're all installed. (This would be a little more complicated, because there are multiple packages, so you'd need to loop over all of them in the `requireNamespace()` check.)

The motivation here is that all of these are only needed for _one function_ and it makes the package a fair amount heavier, so it would be nice for people to _not_ need to install all of these, unless they plan to actually use `v()`.

Reply:
Yeah I agree with this. `shinydashboardPlus` does something [similar](https://github.com/RinteRface/shinydashboardPlus/blob/master/R/shinydashboardPlusGallery.R#L16-L25) for their gallery app. `pkgr` will install them the `Suggests` by default, so this would likely only impact installations outside metrum.

Reply:
@seth127 we are going to make an issue about this